tasks/prescribe_tasks.csv

```diff
@@ -0,0 +1,3 @@
+StartDate,EndDate,IpFile,CostFile,OutputFile
+2020-12-22,2021-06-19,~/covid-xprize-robotasks-phase2/ips/live/20210112_080002_20201222_20210619_ips.csv,~/covid-xprize-robotasks-phase2/ip_costs/fixed_equal_costs.csv,~/work/prescriptions/20210112_080002_robojudge_live.csv
+2020-12-23,2021-06-01,~/covid-xprize-robotasks-phase2/ips/live/20210112_080002_20201222_20210619_ips.csv,~/covid-xprize-robotasks-phase2/ip_costs/fixed_equal_costs.csv,~/work/prescriptions/20210112_080002_robojudge_live2.csv
```

Comment:
We'll need to update the tasks before the sandboxes are live. Running these 2 tasks would be way too slow I believe.

Reply:
Yes these were just samples for testing (though they run ultra quick with the blind-greedy prescriptor)